src/odemis/gui/cont/temperature.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+
+"""
+@author: Anders Muskens
+
+Copyright © 2020 Anders Muskens, Delmic
+
+Handles the switch of the content of the main GUI tabs.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+
+import wx
+import collections
+import numpy
+import time
+import logging
+from odemis.gui.util import call_in_wx_main
+import odemis.gui as gui
+from odemis.gui.comp import popup
+import odemis.model as md
+
+MIN_PERIOD_WARNING = 60  # in seconds, represents the time before notifications are shown again
+
+
+@call_in_wx_main
+def set_label_colour(label, colour, bold=False):
```

Comment:
Now that I think of it, you could just have an argument `weight`, instead of `bold`. Then the caller would call with either wx.BOLD or wx.NORMAL. You wouldn't need the if, and you could avoid the explicit argument name in the callers as it'd be more obvious what we are talking about. (Granted, this is absolute nitpicking!)